images/dns-init/dns-init.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+if [ "$APPEND" == "true" ]; then
+    echo $SEARCH >> /etc/resolv.conf
+else
+    echo -e "$SEARCH\n$(cat /etc/resolv.conf)" > /etc/resolv.conf
```

Comment:
Won't we endup having 2 search lines here?

Reply:
Also we should check if the domain already exists in /etc/resolv.conf before appending